eng/pipelines/templates/stages/archetype-js-release.yml

```diff
@@ -131,12 +131,10 @@ stages:
                             - sdk/**/*.md
                             - .github/CODEOWNERS
                       - download: current
-                      # Pull and build the docker image.
-                      - template: /eng/common/pipelines/templates/steps/docker-pull-image.yml
-                        parameters:
-                          ContainerRegistryClientId: $(azuresdkimages-cr-clientid)
-                          ContainerRegistryClientSecret: $(azuresdkimages-cr-clientsecret)
-                          ImageId: "$(DocValidationImageId)"
+                      - script: |
+                          npm install type2docfx -g
```

Comment:
Should we do this install step in our helper ps1 script instead of in an individual step in all the pipelines?